sig-governance.md

```diff
@@ -0,0 +1,179 @@
+Subproject contributors (as applicable).
+
+### Chair
+
+- SIG Chairs
+  - Establish new subprojects
+  - Decommission existing subprojects
+  - Resolve X-Subproject technical issues and decisions
+  - Number: 2-3
+  - Membership tracked in [sigs.yaml](./sigs.yaml)
+
+### Subproject Owner
+
+- Subproject Owners
+  - Scoped to a subproject defined in [sigs.yaml](./sigs.yaml)
+  - Seed leads and contributors established at subproject founding
+  - *SHOULD* be an escalation point for technical discussions and decisions in
+  the subproject
+  - *SHOULD* set milestone priorities or delegate this responsibility
+  - Number: 2-3
+  - Membership tracked in [sigs.yaml](./sigs.yaml)
+
+### All Leads
+
+- *SHOULD* maintain health of at least one subproject or the health of the SIG.
+- *SHOULD* show sustained contributions to at least one subproject or to the
+  SIG.
+- *SHOULD* hold some documented role or responsibility in the SIG and / or at
+  least one subproject
+    (e.g. reviewer, approver, etc).
+- *MAY* build new functionality for subprojects.
+- *MAY* participate in decision making for the subprojects in which they hold roles.
+- Includes all reviewers and approvers in OWNERS files for subprojects.
+- *SHOULD* take an [Inclusive Open Source Community Orientation course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/) in support of our community values
+within 30 days from the date of their appointment.
+
+### Other Roles
+This governance document outlines the required roles for SIGs.  However, SIGs are allowed to operate how they see fit outside of minimum 
+governance requirements, including defining more roles to sustain the group. If 
+a SIG needs to change the Chair position to include or remove
+duties, this needs to be approved by the Steering Committee. Newly created roles
+that don't assume any responsibility of Chair should follow
+the governing processes in the SIGs charter. 
+
+Example of SIG roles created to help operations:
+
+- The Release Team: Bug Triage, CI Signal, and more  
+- API Reviewer and Moderator  
+- Production Readiness Reviewer 
+- PR Wrangler
+
+Other roles...
+- *MUST* be tracked on the SIGs README with a link to the role definition.
+- *MUST* have the Steering Committees approval to proceed with roles that assume
+duties from Chairs on a non-temporary basis. 
+- *SHOULD* be documented in SIG charters if the role has delegation away from a
+sig-governance.md listed role.
+- *SHOULD* be sent to [contributors@lists.opendatahub.io](mailto:contributors@lists.opendatahub.io) for awareness as a notice 
+and a lazy consensus period when they are newly created.
+- *MAY* Fill in for another named role on a temporary basis.
+
+#### Subproject Creation
+
+Option 1: by SIG Chairs
+
+- Subprojects may be created by socializing the proposal to the corresponding SIG with fallback on majority vote of SIG Chairs.  The result *SHOULD* be supported by the majority of SIG Leads.
```

Comment:
Here `SIG Leads` refers to _all_ SIG leads or _just_ the leads for the specific SIG?

Reply:
Refers to the leads for the specific SIG.  I've added language to make that more clear.